pandera/api/dataframe/model_components.py

```diff
@@ -320,3 +349,40 @@ def _wrapper(fn: Union[classmethod, AnyCallable]) -> classmethod:
     if _fn:
         return _wrapper(_fn)  # type: ignore
     return _wrapper
+
+
+ClassParser = Callable[[Union[classmethod, AnyCallable]], classmethod]
+
+
+def parse(*fields, **parse_kwargs) -> ClassParser:
```

Comment:
should this be `parser`? I feel like verb tense `parse` also makes sense, but my sense is it should just be the lowercase form of the `Parser` object.

Reply:
I am fine with either, so I respect your opinion.
Fixed